site/content/blogs/release-v127.en.md

```diff
@@ -38,14 +38,129 @@ or multiple listeners on different ports and protocols for [network load balance
 
 ## Extend Copilot-generated AWS CloudFormation templates
 
-#### Preview AWS CloudFormation template changes
+AWS Copilot enables builders to quickly get started with containerized applications through the `copilot init` command and following prompts.
+Developers can then grow their applications by editing and deploying [manifest](../docs/manifest/overview.en.md) infrastructure-as-code files.
+And now with v1.27, developers can extend any Copilot-generated CloudFormation template with the `copilot [noun] override` command, enabling them to fully customize their infrastructure.
+
+### AWS Cloud Development Kit (CDK) Overrides
+
+You can use the CDK to extend your CloudFormation templates when you need the expressive power and safety of a
+programming language. After running the `copilot [noun] override` command, Copilot will generate a CDK application
+under the `copilot/[name]/override` directory:
+
+```console
+.
+├── bin/
+│   └── override.ts
+├── .gitignore
+├── cdk.json
+├── package.json
+├── README.md
+├── stack.ts
+└── tsconfig.json
+```
+
+You can get started with adding, removing, or replacing properties by editing the `stack.ts` file.
+
+??? note "View sample `stack.ts`"
+
+    ```typescript
+    import * as cdk from 'aws-cdk-lib';
+    import * as path from 'path';
+    import { aws_elasticloadbalancingv2 as elbv2 } from 'aws-cdk-lib';
+    import { aws_ec2 as ec2 } from 'aws-cdk-lib';
+    
+    interface TransformedStackProps extends cdk.StackProps {
+        readonly appName: string;
+        readonly envName: string;
+    }
+    
+    export class TransformedStack extends cdk.Stack {
+        public readonly template: cdk.cloudformation_include.CfnInclude;
+        public readonly appName: string;
+        public readonly envName: string;
+    
+        constructor (scope: cdk.App, id: string, props: TransformedStackProps) {
+            super(scope, id, props);
+            this.template = new cdk.cloudformation_include.CfnInclude(this, 'Template', {
+                templateFile: path.join('.build', 'in.yml'),
+            });
+            this.appName = props.appName;
+            this.envName = props.envName;
+            this.transformPublicNetworkLoadBalancer();
+        }
+    
+        /**
+         * transformPublicNetworkLoadBalancer removes the "Subnets" properties from the NLB,
+         * and adds a SubnetMappings with predefined elastic IP addresses.
+         */
+        transformPublicNetworkLoadBalancer() {
+            const elasticIPs = [new ec2.CfnEIP(this, 'ElasticIP1'), new ec2.CfnEIP(this, 'ElasticIP2')];
+            const publicSubnets = cdk.Fn.importValue(`${this.appName}-${this.envName}-PublicSubnets`);
+    
+            // Apply the override.
+            const nlb = this.template.getResource("PublicNetworkLoadBalancer") as elbv2.CfnLoadBalancer;
+            nlb.addDeletionOverride('Properties.Subnets');
```

Comment:
Does it make sense to link https://docs.aws.amazon.com/cdk/v2/guide/cfn_layer.html or similar somewhere in the docs to help them better write their override code?